qm-plugins/qm-cron/class-qm-cron-output-html.php

```diff
@@ -0,0 +1,333 @@
+						?>
+					</td>
+					<td><?php echo esc_html( gmdate( 'H:i:s' ) ); ?></td>
+				</tr>
+			</tbody>
+		</table>
+		<br>
+		<h3><b>Schedules</b></h3>
+		<table id="qm-cron-schedules" style="width: 100%">
+			<thead>
+				<tr>
+					<th><?php esc_html_e( 'Interval Hook', 'query-monitor' ); ?></span></th>
+					<th><?php esc_html_e( 'Interval (S)', 'query-monitor' ); ?></th>
+					<th><?php esc_html_e( 'Interval (M)', 'query-monitor' ); ?></th>
+					<th><?php esc_html_e( 'Interval (H)', 'query-monitor' ); ?></th>
+					<th><?php esc_html_e( 'Display Name', 'query-monitor' ); ?></th>
+				</tr>
+			</thead>
+			<tbody>
+				<?php
+				foreach ( $data['schedules'] as $schedule => $value ) {
+					echo '<tr>';
+					echo '<th>' . esc_html( $schedule ) . '</th>';
+					echo '<th>' . esc_html( $value['interval'] ) . '</th>';
+					echo '<th>' . esc_html( $this->get_minutes( $value['interval'] ) ) . '</th>';
+					echo '<th>' . esc_html( $this->get_hours( $value['interval'] ) ) . '</th>';
+					echo '<th>' . esc_html( $value['display'] ) . '</th>';
+					echo '</tr>';
+				}
+				?>
+			</tbody>
+		</table>
+		<br>
+		<h3><b>Custom Events</b></h3>
+		<?php $this->display_events( $data['user_crons'], 'qm-cron-user-crons' ); ?>
+		<br>
+		<h3><b>Core Events</b></h3>
+		<?php $this->display_events( $data['core_crons'], 'qm-cron-core-crons' ); ?>
+		</div>
+		<?php
+	}
+
+	/**
+	 * Displays the events in an easy to read table.
+	 *
+	 * @param array  $events        Array of events.
+	 * @param string $no_events_msg Message to display if there are no events.
+	 */
+	private function display_events( $events, $html_table_id = '', $no_events_msg = 'No events found' ) {
+		// Exit early if no events found.
+		if ( ! is_array( $events ) || empty( $events ) ) {
+			echo '
+			<p>', esc_html( $no_events_msg ), '</p>';
+			return;
+		}
+
+		echo '
+			<table class="qm-cron-table qm-cron-event-table">
+				<thead><tr>
+					<th class="col1">', esc_html__( 'Next Execution', 'query-monitor' ), '</th>
+					<th class="col2">', esc_html__( 'Hook', 'query-monitor' ), '</th>
+					<th class="col3">', esc_html__( 'Interval Hook', 'query-monitor' ), '</th>
+					<th class="col4">', esc_html__( 'Interval Value', 'query-monitor' ), '</th>
+					<th class="col5">', esc_html__( 'Args', 'query-monitor' ), '</th>
+				</tr></thead>
+				<tbody>';
+
+		foreach ( $events as $time => $time_cron_array ) {
+			$time        = (int) $time;
+			$event_count = $this->get_arg_set_count( $time_cron_array );
+			$show_time   = true;
+
+			foreach ( $time_cron_array as $hook => $data ) {
+				$row_attributes = $this->get_event_row_attributes( $time, $hook );
+				$arg_set_count  = count( $data );
+				$show_hook      = true;
+
+				foreach ( $data as $hash => $info ) {
+					echo '<tr', $row_attributes, '>'; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+
+					if ( true === $show_time ) {
```

Comment:
```suggestion
					if ( $show_time ) {
```

`$show_time` and `$show_hook` can only be `true` or `false`; we can probably remove the comparison.